examples/custom_issue_manager.py

```diff
@@ -16,39 +18,55 @@ class CustomIssueManager(IssueManager):
     CleanVision can simultaneously check your data for alongside its built-in issue types.
     """
 
-    issue_name = ISSUE_NAME
-    visualization = "property_based"
+    issue_name: str = ISSUE_NAME
+    visualization: str = "property_based"
 
-    def __init__(self, params):
+    def __init__(self) -> None:
         super().__init__()
-        self.set_params(params)
+        self.params = self.get_default_params()
 
-    def get_default_params(self):
+    def get_default_params(self) -> Dict[str, Any]:
         return {"threshold": 0.4}
 
-    def set_params(self, params):
+    def update_params(self, params: Dict[str, Any]) -> None:
         self.params = self.get_default_params()
         non_none_params = {k: v for k, v in params.items() if v is not None}
         self.params = {**self.params, **non_none_params}
 
     @staticmethod
-    def calculate_mean_pixel_value(image):
+    def calculate_mean_pixel_value(image: Image.Image) -> float:
         gray_image = image.convert("L")
         return np.mean(np.array(gray_image))
 
-    def get_scores(self, raw_scores):
+    def get_scores(self, raw_scores: "np.ndarray[Any, Any]") -> "np.ndarray[Any, Any]":
         scores = np.array(raw_scores)
         return scores / 255.0
 
-    def mark_issue(self, scores, threshold):
+    def mark_issue(
+        self, scores: "np.ndarray[Any, Any]", threshold: float
+    ) -> "np.ndarray[Any, Any]":
```

Comment:
The output of `get_scores` is provided as `scores` in this method. So we can just use the same type hint.
The element-wise comparisons return an array (vector?) of booleans.

```suggestion
        self, scores: npt.NDArray[np.float_], threshold: float
    ) -> npt.NDArray[np.bool_]:
```